docs/tutorial/rdd.md

```diff
@@ -181,7 +181,7 @@ We use [checkin.csv CSV file](#pointrdd-from-csvtsv) as the example. You can cre
 
 1. Load data in GeoSparkSQL.
 ```Scala
-var df = sparkSession.read.format("csv").option("delimiter", "\t").option("header", "false").load(csvPointInputLocation)
+var df = sparkSession.read.format("csv").option("header", "false").load(csvPointInputLocation)
```

Comment:
Can you keep the delimiter option here?

Reply:
Thank you for the review! Just before this example, the text says "We use checkin.csv CSV file as the example", so I thought this example should be coordinated with it.
How about keeping the delimiter option in the example, and modifying the sentence like "We use checkin.csv CSV file with tab delimiters as the example"?

Reply:
@sergiimk Sorry, I misunderstood the example. I think what you did is correct. I will merge this PR.